superset-frontend/src/SqlLab/components/TabbedSqlEditors.jsx

```diff
@@ -289,9 +290,13 @@ class TabbedSqlEditors extends React.PureComponent {
       const title = (
         <>
           {qe.title} <TabStatusIcon tabState={state} />{' '}
-          <span className="close" onClick={() => this.removeQueryEditor(qe)}>
-            {'×'}
-          </span>
+          <Icon
+            name="cancel-x"
+            height="16px"
+            width="16px"
```

Comment:
I'm surprised the pixel strings are required here, did you test to see if numbers work?

Reply:
It turn out all these extra props are not necessary...i thought new icon pushed the tab taller but actually not. new svg (with 24 x 14) is good to use.